site/app/models/RainbowCustomizationJSON.php

```diff
@@ -28,20 +28,14 @@ class RainbowCustomizationJSON extends AbstractModel {
     private $gradeables = [];
 
     // The order of allowed_display and allowed_display_description has to match
-    const allowed_display = ['grade_summary', 'grade_details', 'benchmark_percent',
-        'exam_seating', 'section', 'messages', 'warning', 'final_grade', 'manual_grade', 'final_cutoff', 'instructor_notes'];
+    const allowed_display = ['grade_summary', 'grade_details',
+        'exam_seating', 'final_grade', 'instructor_notes'];
```

Comment:
If you format arrays such that every field is on a separate line, the diff is much easier to read when changes are made in the future.  Not a requested change--just a general suggestion.
```suggestion
    const allowed_display = [
        'grade_summary',
        'grade_details',
        'exam_seating',
        'final_grade',
        'instructor_notes',
    ];
```